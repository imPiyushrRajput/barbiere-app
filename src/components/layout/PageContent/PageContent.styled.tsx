import { styled } from '@styled';

export const PageContent = styled('main')`
    background-color: #eff2f4;
    overflow: auto;
    padding-top: 32px;

    @media (max-width: 1279px) {
        padding-left: 24px;
        padding-right: 24px;
    }

    @media (min-width: 1280px) {
        padding-left: calc(24px + (((100vw - 1280px) / 160) * 16));
        padding-right: calc(24px + (((100vw - 1280px) / 160) * 16));
    }

    @media (min-width: 1440px) {
        padding-left: 40px;
        padding-right: 40px;
    }
`;
