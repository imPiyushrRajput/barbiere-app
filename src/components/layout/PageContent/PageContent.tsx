import { WithStyle } from '@medly-components/utils';
import React from 'react';
import * as Styled from './PageContent.styled';

export const PageContent: React.SFC & WithStyle = props => {
    return <Styled.PageContent {...props}>{props.children}</Styled.PageContent>;
};
PageContent.displayName = 'PageContent';
PageContent.Style = Styled.PageContent;
