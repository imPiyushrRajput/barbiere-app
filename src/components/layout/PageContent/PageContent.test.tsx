import { render } from '@test-utils';
import React from 'react';
import { PageContent } from './PageContent';

describe('PageContent component', () => {
    it('should render properly', () => {
        const { container } = render(<PageContent>Demo PageContent</PageContent>);
        expect(container).toMatchSnapshot();
    });
});
