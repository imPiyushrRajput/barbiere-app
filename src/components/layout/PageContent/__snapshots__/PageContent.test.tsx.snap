// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageContent component should render properly 1`] = `
.c0 {
  background-color: #eff2f4;
  overflow: auto;
  padding-top: 32px;
}

@media (max-width:1279px) {
  .c0 {
    padding-left: 24px;
    padding-right: 24px;
  }
}

@media (min-width:1280px) {
  .c0 {
    padding-left: calc(24px + (((100vw - 1280px) / 160) * 16));
    padding-right: calc(24px + (((100vw - 1280px) / 160) * 16));
  }
}

@media (min-width:1440px) {
  .c0 {
    padding-left: 40px;
    padding-right: 40px;
  }
}

<div>
  <main
    class="c0"
  >
    Demo PageContent
  </main>
</div>
`;
